Package.swift

```diff
@@ -46,6 +56,23 @@ let package = Package(
                     resources: [
                         .copy("Resources/single_add.basic.ort")
                     ]),
+        .target(name: "OnnxRuntimeExtensions",
+                dependencies: ["onnxruntime_extensions", "onnxruntime"],
+                path: "extensions",
+                cxxSettings: [
+                    .define("SPM_BUILD"),
```

Comment:
The macro definition name may be updated.

Reply:
Can this be updated now?